spec/classes/minecraft_source_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe 'minecraft::source', :type => :class do
+  let :pre_condition do
+    'include ::minecraft'
+  end
+
+  let :facts do
+    {
+      :osfamily               => 'debian',
+      :operatingsystem        => 'ubuntu',
+      :operatingsystemrelease => '14.04',
+      :lsbdistcodename        => 'trusty',
+      :id                     => 'root',
+      :concat_basedir         => '/dne',
+      :path                   => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
+      :kernel                 => 'Linux',
+    }
+  end
+
+  it { is_expected.to compile }
```

Comment:
`it { is_expected.to compile.with_all_deps }` *can* be better.  It makes sure you're not referencing resources that don't exist.